pipeline.wdl

```diff
@@ -0,0 +1,85 @@
+version 1.0
+
+import "tasks/common.wdl" as common
+import "tasks/delly.wdl" as delly
+import "tasks/manta.wdl" as manta
+import "tasks/picard.wdl" as picard
+import "tasks/bcftools.wdl" as bcftools
+import "tasks/survivor.wdl" as survivor
+import "tasks/clever.wdl" as clever
+import "tasks/bwa.wdl" as bwa
+
+workflow SVcalling {
+    input {
+        IndexedBamFile bamFile
+        Reference reference
+        BwaIndex bwaIndex
+        String sample
+        String outputDir
```

Comment:
It is customary nowadays to add a deafult of `"."` to output directory inputs.

Reply:
Done